dsas-web-ui/src/main/java/gov/usgs/cida/dsas/dao/shoreline/ShorelineLidarFileDAO.java

```
@@ -41,20 +47,44 @@
  */
 public class ShorelineLidarFileDAO extends ShorelineFileDAO {
 
+	private static final Logger log = LoggerFactory.getLogger(ShorelineLidarFileDAO.class);
+	
+	private static final int BATCH_SIZE = 5000;
+	
+	private CoordinateReferenceSystem targetCRS;
+	
 	public ShorelineLidarFileDAO() {
 		this.JNDI_NAME = PropertyUtil.getProperty(Property.JDBC_NAME);
+		// Workaround for lat/lon vs lon/lat in reprojection
+		CRSAuthorityFactory   factory = CRS.getAuthorityFactory(true);
+		try {
+			this.targetCRS = factory.createCoordinateReferenceSystem("EPSG:4326");
+		} catch (FactoryException ex) {
+			// Something really bad is happening
+			this.targetCRS = null;
+			log.error("Unable to get CRS for WGS84");
```

We probably want to rethrow this exception instead of logging and swallowing. This would not be a recoverable event